WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderSearchSuggestionListeners.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.ui.reader.adapters
+
+interface OnSuggestionClickListener {
+    fun onSuggestionClicked(query: String?)
+}
```

Comment:
AFAIU it seems we are not currently using this interface, maybe worth considering removing it? wdyt?

Reply:
That's actually used by `ReaderSearchSuggestionRecyclerAdapter`. It's just not used by the existing `ReaderSearchSuggestionAdapter`, which uses `SearchView`'s own `OnSuggestionListener` for that.

Reply:
Yep I see! Sorry I was missing it 😊!